docs/content/en/changelog/changelog.md

```diff
@@ -7,6 +7,13 @@ Here are the release notes for **DefectDojo Pro (Cloud Version)**. These release
 
 For Open Source release notes, please see the [Releases page on GitHub](https://github.com/DefectDojo/django-DefectDojo/releases), or alternatively consult the Open Source [upgrade notes](../../open_source/upgrading/upgrading_guide).
 
+## Dec 31, 2024: v2.41.4
+
+- **(API)** Changed functionality of 'Force To Active / Verified' flag: True now forces to Active, while False will keep the tool's status (rather than forcing to Inactive).
```

Comment:
I believe `None` will keep the default of the tool/parser. `False` will override it to be `False` still. In the PR I see that only the `help_text` was changed, but not the functionality. Or am I missing a PR somewhere :-)

That same PR updated the `help_text` for `Verified` flag as well.

https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/importers/default_reimporter.py#L650-L657